main.m

```diff
@@ -29,10 +29,25 @@
  */
 
 #import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
 #import <Security/Security.h>
+#import "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Versions/C/Headers/NSTask.h"
 #import "sqlite3.h"
+#include "stdio.h"
 
-void printToStdOut(NSString *format, ...) {
+#define KNRM  "\x1B[0m"
+#define KRED  "\x1B[31m"
+#define KGRN  "\x1B[32m"
+#define KYEL  "\x1B[33m"
+#define KBLU  "\x1B[34m"
+#define KMAG  "\x1B[35m"
+#define KCYN  "\x1B[36m"
+#define KWHT  "\x1B[37m"
+
+NSString *SELECTEDENTITLEMENT = @"none";
```

Comment:
I'm admittedly not an objective-c expert, but this looked surprising to me. We basically want a static file-local global right? Would this be something more like `static NSString *selecedEntitelment = @"none";`?

Reply:
Me neither, but from what I learned at python coding, global variable names should be capitalized :). This may not be applicable in objective-c.

Reply:
Yeah..every language has their own patterns. Also, you do t want a “global variable” so much as a file-local variable.  Let’s stick with what objective-c folks do. Can you see if what I pasted above works?

Reply:
Adjusted the name, according to your recommendation.